bundles/net.tourbook.device/src/net/tourbook/device/sporttracks/FitLogSAXHandler.java

```diff
@@ -847,7 +855,7 @@ private void finalizeTour_20_SetTags(final TourData tourData) {
       boolean searchTagById = false;
```

Comment:
There are 3 locations with `size() == 0` instead of the old code `isEmpty()`

Reply:
ok I will correct that